prov/tcp/src/tcpx_ep.c

```diff
@@ -185,6 +200,42 @@ static int tcpx_ep_shutdown(struct fid_ep *ep, uint64_t flags)
 	return ret;
 }
 
+static int tcpx_pep_bind_to_port_range(struct tcpx_pep *pep)
+{
+	int ret, i, rand_port_number;
+
+	srand(getpid());
```

Comment:
sand() requires including stdlib.h.  Please add that as an include file.
We don't want to call srand() on every call into this function.  That will reset the random number generator to the same starting value each time.  Move srand() into the tcp initialization function.